infrastructure/src/main/java/org/corfudb/infrastructure/IServerRouter.java

```diff
@@ -168,6 +169,12 @@ default boolean validateRequest(RequestMsg req, ChannelHandlerContext ctx) {
             return true;
         }
 
+        // TODO: In some cases this needs to be handled specially in the future
```

Comment:
Could you please help me understand more as to in which cases we need to handle this especially?  Do you mean specific handling(throw error/print and return something) in the case of a version mismatch?

Reply:
I thought currently this is mainly used for logging and as a placeholder. From our discussion before, I remember we want to forward requestMsg with different versions to different versions of handlers/servers (for example, `Sequencer ServerV2` or `handleTokenRequestV2`, this part of design is still pending to be determined though)

Reply:
Note: We finally decide to remove this logging here to avoid flooding the logs.